src/schema/rules/files/raw/task.yaml

```diff
@@ -37,6 +37,7 @@ timeseries:
     acquisition: optional
     run: optional
     recording: optional
+    modality: optional
```

Comment:
I think this needs to go on the `timeseries__anat` rule below, to achieve what you're trying to accomplish.